ansible_base/jwt_consumer/hub/auth.py

```diff
@@ -51,25 +58,38 @@ def process_permissions(self):
                         team = Resource.objects.get(ansible_id=ansible_id).content_object
                     except Resource.DoesNotExist:
                         team = self.common_auth.get_or_create_resource('team', team_data)[1]
-                    teams.append(team)
 
-                    groups.append(team.group)
+                groups.append(team.group)
+
+                if role_name == 'Team Member':
+                    member_teams.append(team)
```

Comment:
Shouldn't this be indented under the `for` loop?

Reply:
yep